packages/react-components/src/components/FileCard.tsx

```diff
@@ -99,31 +128,40 @@ export const _FileCard = (props: _FileCardProps): JSX.Element => {
   };
 
   return (
-    <Stack
-      className={containerClassName}
-      onClick={() => {
-        props.actionHandler && props.actionHandler();
-      }}
-    >
-      <Stack horizontal horizontalAlign="space-between" verticalAlign="center" className={fileInfoWrapperClassName}>
-        <Stack>
-          {/* We are not using <ChatCompositeIcon /> here as we currently do not support customizing these filetype icons. */}
-          <Icon
-            {...getFileTypeIconProps({
-              extension: fileExtension,
-              size: 24,
-              imageFileType: 'svg'
-            })}
-          />
-        </Stack>
-        <Stack className={fileNameContainerClassName}>
-          <Text className={fileNameTextClassName}>{fileName}</Text>
-        </Stack>
-        <Stack verticalAlign="center" className={actionIconClassName}>
-          {actionIcon && actionIcon}
+    <>
+      <Announcer announcementString={announcerString} ariaLive={'polite'} />
+      <Stack
+        className={containerClassName}
+        onClick={() => {
+          props.actionHandler && props.actionHandler();
+        }}
+      >
+        <Stack horizontal horizontalAlign="space-between" verticalAlign="center" className={fileInfoWrapperClassName}>
+          <Stack>
+            {/* We are not using <ChatCompositeIcon /> here as we currently do not support customizing these filetype icons. */}
+            <Icon
+              {...getFileTypeIconProps({
+                extension: fileExtension,
+                size: 24,
+                imageFileType: 'svg'
+              })}
+            />
+          </Stack>
+          <Stack className={fileNameContainerClassName}>
+            <Text className={fileNameTextClassName}>{fileName}</Text>
+          </Stack>
+          <Stack verticalAlign="center" className={actionIconClassName}>
+            {actionIcon && actionIcon}
+          </Stack>
         </Stack>
+        {showProgressIndicator && (
+          <ProgressIndicator
+            percentComplete={progress}
+            styles={progressIndicatorStyles}
+            ariaValueText={`${fileName} ${localeStrings.uploadProgress}`}
```

Comment:
This ideally should not contain the filename.
This prop is used only to depict the progress value as per Fluent docs.
For example, if value is 0.25, the string here should be 25%

Reply:
yes the VoiceOver already announces the percentage, removing the customized aria label here